code/extensions/che-remote/package.nls.json

```diff
@@ -3,5 +3,6 @@
 	"description": "Provides remoteIndicator commands",
 	"openDocumentationCommand": "Eclipse Che: Open Documentation",
 	"openDashboardCommand": "Eclipse Che: Open Dashboard",
-	"stopWorkspaceCommand": "Eclipse Che: Stop Workspace"
+	"stopWorkspaceCommand": "Eclipse Che: Stop Workspace",
+	"restartWorkspaceCommand": "Eclipse Che: Restart Workspace"
```

Comment:
The corresponding item should be added to [this section](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/branding/product.json#L21) to provided branding for the `devspaces-code`

Reply:
And to [the Branding doc](https://github.com/che-incubator/che-code/blob/main/README.md#branding-the-ui) as well :wink:

Reply:
The PR to the `devspaces-code`: https://github.com/redhat-developer/devspaces-images/pull/397

Reply:
The PR to the README file: https://github.com/che-incubator/che-code/pull/177